lab_SOPs/labmeetings.md

```diff
@@ -0,0 +1,17 @@
+# VKC Lab Meetings
+
+The VKC Lab meets once a week during the fall and spring semesters. Time and location varies (see [below](#schedule)).
+
+Generally, [an agenda](#agendas) is shared on the `#general` channel on the `VKC-Lab` Slack at the beginning of each week. We discuss lab business and one or two lab members give a 10-15 minute [presentation](#presentations) on their work.
+
+## Schedule
+
+The schedule for lab meetings can be found in [this Airtable view](https://airtable.com/shrhkxQgCqwWbmQvW). It can also be [exported as a calendar format](https://support.airtable.com/hc/en-us/articles/225050348-Integrating-Airtable-with-external-calendar-applications-e-g-Google-Calendar-Microsoft-Outlook-Apple-Calendar-) which can be added to iCal or Google Calendar.
+
+## Agendas
+
+All lab agendas can be found in `VKC LAB > Lab Meetings > Agendas`. Starting in spring 2020, agendas for each semester will be organized in one running document labeled `S/F YYYY Agenda`.
```

Comment:
Can we do `YYYY S/F` instead? For things like this, consider how sorting affects the way we search for things? Semantically, it doesn't really make sense for all fall semesters to be grouped.

And maybe `Spring/Fall` in case someday we want to have `Summer` too (that would cause a collision with `S`)